src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -116,7 +131,32 @@ public void printBoard(){
         }
     }
 
-    public Player getCurrentPlayer() {
+    public void scoreBoard(){
```

Comment:
Needs a more descriptive name. Something like "updateScore".

Reply:
I see @pennyfea has fixed this. Can you leave comments on the requested changes when you have completed them? Thanks!

Reply:
Resolved  by  [ed240d7](https://github.com/Perplex/tictactoe/commit/ed240d7eff9f7396828639c06fc2c889eb71482b)